TestingLowerBounds/ForMathlib/EReal.lean

```diff
@@ -240,3 +240,25 @@ instance : MeasurableMul₂ EReal := by
   sorry
 
 end EReal
+
+section ENNReal
```

Comment:
That section should be closed at the end of the file. But since the namespace below also plays the role of a section, we can actually delete it.
```suggestion
```